bot/exts/evergreen/catify.py

```diff
@@ -0,0 +1,44 @@
+import random
+from typing import Optional
+
+from discord.ext import commands
+
+from bot.constants import Cats
+
+
+class Catify(commands.Cog):
+    """Cog for the catify command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["ᓚᘏᗢify", "ᓚᘏᗢ"])
+    async def catify(self, ctx: commands.Context, *string: Optional[str]) -> None:
+        """Catifies a string or username."""
+        if string == ():
+            username = ctx.author.name
+            if len(username) >= 28:
+                return await ctx.send("Your username is too long to be catified! Please change it.")
+            else:
+                username += f" | {random.choice(Cats.cats)}"
+                await ctx.send(f"Your catified username is: `{username}`")
+                await ctx.author.edit(nick=username)
```

Comment:
I don't think we should edit the nickname here.

Reply:
why, thats part of the command

Reply:
thats intended